package.json

```diff
@@ -19,7 +19,8 @@
   },
   "homepage": "https://github.com/majac91/Travel_journal#readme",
   "dependencies": {
-    "moment": "^2.27.0"
+    "moment": "^2.27.0",
+    "src": "^1.1.2"
```

Comment:
Why did you need to install `src`?

Reply:
Moment was in the npm tutorial you’ve sent me I think, I followed along

Reply:
Or maybe  it was in the documentation for moment, I’m not home to check rn

Reply:
Just checked the video  and it's not a part of the tutorial but it's also not something I remember installing separately. Didn't it come with other npm modules?

Reply:
I guess you did `npm install src` without realizing it. It's the only explanation. You can uninstall that dependency with `npm uninstall src` :)